ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/LoadDatabase.java

```diff
@@ -162,6 +184,10 @@ private boolean isDbEmpty(String dbName) throws HiveException {
     Map<String, String> parameters = new HashMap<>(dbObj.getParameters());
     parameters.remove(ReplicationSpec.KEY.CURR_STATE_ID.toString());
 
+    parameters.remove(ReplUtils.REPL_IS_CUSTOM_DB_LOC);
```

Comment:
why are these params needed?

Reply:
These parameters are passed from dump to signify  the database has custom locations on source  or not.